shell/common/api/electron_api_url_loader.cc

```diff
@@ -462,6 +469,10 @@ void SimpleURLLoaderWrapper::Cancel() {
 }
 scoped_refptr<network::SharedURLLoaderFactory>
 SimpleURLLoaderWrapper::GetURLLoaderFactoryForURL(const GURL& url) {
+  if (electron::IsUtilityProcess()) {
+    return URLLoaderBundle::GetInstance()->GetSharedURLLoaderFactory();
```

Comment:
Yes, that's correct. It depends on the browser session. Similarly, `net.resolveHost` is also not supported since it's implemented in the session. I'll add a section to the documentation for what's different in the utility process.

Reply:
Done in 53bb415